src/debug/webgl-debug.js

```diff
@@ -152,7 +152,126 @@ var glValidEnumContexts = {
 
   // EXT_blend_minmax extension
 
-  'blendEquationEXT': {1: { 0:true }}
+  'blendEquationEXT': {1: { 0:true }},
+
+  // WebGL 2 Buffer objects
+
+  'bufferData': {3: { 0:true, 2:true }, 4: { 0:true, 2:true }, 5: { 0:true, 2:true }},
```

Comment:
Could you add comments indicating which entries for bufferData and bufferSubData are for WebGL 1.0 vs. 2.0?